apps/framework-cli/src/main.rs

```diff
@@ -14,14 +14,16 @@ fn main() {
         "production"
     };
 
-    let _guard = sentry::init(
-        ("https://83941f36fe439ebe1ffd4d35b834ed7a@o4505851966128128.ingest.sentry.io/4505851967963136", 
-        sentry::ClientOptions {
-        release: sentry::release_name!(),
-        traces_sample_rate: 1.0,
-        environment: Some(envionment.into()),
-        ..Default::default()
-      }));
+    if envionment == "production" {
+        let _guard = sentry::init(
+            ("https://83941f36fe439ebe1ffd4d35b834ed7a@o4505851966128128.ingest.sentry.io/4505851967963136", 
```

Comment:
Is exposing this URL a security issue? Could a bad actor mess with our metric?

Reply:
Yeah - I have been thinking about that. Unfortunatly this is software that runs anonymously on people's laptop. As such we need to be able to get metrics anonymously. We could make it a bit harder by embedding it at build time. But if someone is motivated, they will be able to get that back from the binary anyway.